specreduce/utils/synth_data.py

```diff
@@ -68,3 +77,234 @@ def make_2dspec_image(
     ccd_im = CCDData(noisy_image, unit=u.count)
 
     return ccd_im
+
+
+def make_2d_arc_image(
+    nx=3000,
+    ny=1000,
+    wcs=None,
+    extent=[3500, 7000],
+    wave_air=False,
+    wave_unit=u.Angstrom,
+    background=5,
+    line_fwhm=5.,
+    linelists=['HeI'],
+    amplitude_scale=1.,
+    tilt_func=None
+):
+    """
+    Create synthetic 2D spectroscopic image of reference emission lines, e.g. a calibration
+    arc lamp. Currently, linelists from ``pypeit`` are supported and are selected by string or
+    list of strings that is passed to `~specreduce.calibration_data.load_pypeit_calibration_lines`.
+    If a ``wcs`` is not provided, one is created using ``extent`` and ``wave_unit`` with
+    dispersion along the X axis.
+
+    Parameters
+    ----------
+    nx : int (default=3000)
+        Size of image in X axis which is assumed to be the dispersion axis
+    ny : int (default=1000)
+        Size of image in Y axis which is assumed to be the spatial axis
+    wcs : `~astropy.wcs.WCS` instance or None (default: None)
+        2D WCS to apply to the image. Must have a spectral axis defined along with
+        appropriate spectral wavelength units.
+    extent : 2-element list-like
+        If ``wcs`` is not provided, this defines the beginning and end wavelengths
+        of the dispersion axis.
+    wave_air : bool (default: False)
+        If True, convert the vacuum wavelengths used by ``pypeit`` to air wavelengths.
```

Comment:
having this between the two arguments that are only applicable if `wcs` is not provided might be confusing - could we move it below `extent` and `wave_unit`?

Reply:
i'll at least do that if there's not a more clever way around it...